cg/meta/workflow/balsamic.py

```diff
@@ -455,7 +495,7 @@ def get_verified_config_case_arguments(
             else None
         )
 
-        return {
+        args_dict = {
```

Comment:
```suggestion
        config_case: Dict[str, Any] = {
```
Or is only str values?

Reply:
Only string values :)